azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -188,6 +188,24 @@ public void alertOnFailedUpdate(final Executor executor, List<ExecutableFlow> fl
     }
   }
 
+  @Override
```

Comment:
Please add a function-level description of what this function does.

Reply:
Done.